.ci/oci-hacbs-test-e2e.sh

```diff
@@ -0,0 +1 @@
+echo "PASS"
```

Comment:
This test data is to onboard openshift CI in our project. since we are moving to a controller mechanism in a different repo. i must close this PR here and create a new one on the newly created repo.